src/runtime_src/core/pcie/emulation/common_em/unix_socket.cxx

```diff
@@ -35,7 +35,10 @@ unix_socket::unix_socket(const std::string& sock_id,double timeout_insec,bool fa
     name = pathname + "/" + socket;
     systemUtil::makeSystemCall(pathname, systemUtil::systemOperation::CREATE);
   } else {
-    name = "/tmp/" + socket;
+    //name = "/tmp/" + socket;
+    name = "/tmp/" + std::string(cUser) + "/" + socket;
```

Comment:
This will crash if cUser is null. Please fix it.

Reply:
Removed the unix socket changes and deferred to Uday to do the required changes